### PR TITLE
Cnmf traces

### PR DESCRIPTION
**Pull request by ArjunPutcha**
Loads up new napari viewer with Napari1D to display the image and relevant traces

### CODE REVIEW

File: caiman_napari/napari1d_manager.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    layer = viewer1d.add_inf_line(data=[1], orientation="vertical", color="red", width=3, name="slider")
+    viewer1d.add_layer(layer=layer)
+
+    # Confirmed the time variable updates real time
+    @viewer.dims.events.current_step.connect
+    def update_slider(event):
+        time = viewer.dims.current_step[0]
+        viewer.text_overlay.text = f"{time:1.1f} time"
+        # Remove most recent layer, recreate new line
+        viewer1d.layers.remove(viewer1d.layers[-1])
```
Comment (kushalkolar) [inline caiman_napari/napari1d_manager.py:80]: try `InfLine.move()`

https://github.com/lukasz-migas/napari-1d/blob/main/napari_plot/layers/infline/infline.py#L247

File: caiman_napari/algorithms/cnmf.py [hunk 2]
```diff
@@ -163,15 +157,26 @@ def load_output(viewer, batch_item: pd.Series):
...
 
-        viewer.add_shapes(
-            data=contours_bad_coordinates,
-            shape_type='polygon',
-            edge_width=0.5,
-            edge_color=colors_contours_bad_edge,
-            face_color=colors_contours_bad_face,
-            opacity=0.7,
-        )
+    napari1d_run(image=Cn, shapes=shapes_dict)
```
[the remark this answers is not in the document]
Comment (ArjunPutcha) [reply caiman_napari/algorithms/cnmf.py]: Yup to both!

[the remark this answers is not in the document]
Comment (kushalkolar) [reply caiman_napari/algorithms/cnmf.py]: Sorry I guess this was an old comment

**Comment by kushalkolar** [comment]
merging this into dev early because I need to do refactor

**Pull request merged**